src/SonarScanner.MSBuild.TFS.Classic/Resources.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!--
-    Microsoft ResX Schema
-
+  <!-- 
+    Microsoft ResX Schema 
+    
```

Comment:
You added all the whitespace back. I think it's better without it.

Reply:
I used the VS designer to edit the resx. And funny enough my Github diffview doesn't show the whitespace:

![image](https://user-images.githubusercontent.com/103252490/203502113-6d894cbf-9ebf-4bc0-9f0c-f34ced3221c1.png)
 
(I check the hide Whitespace setting. It is off).

Reply:
Strange. It seems to be a bug somewhere

Reply:
You can check https://github.com/SonarSource/sonar-scanner-msbuild/pull/1362/commits/4597c1b037c4e417e6c55c39d5d71f80aaaa8574

Reply:
I think it was this way:
Before: Additional whitespace was there because the last edit was done using the VS UI for resx files.

* I changed the message by editing the resx with the VS XML editor -> the XML got formatted/cleaned up and the whitespace disappeared adf1ac8f3a53ef65619fd401bf840ec973a689dc
* I changed the message again using the VS  UI 4597c1b037c4e417e6c55c39d5d71f80aaaa8574 the whitespace was reintroduced by the UI

The final diff has the additional whitespace and therefore only shows the actual changes.